src/partners/partners.json

```diff
@@ -255,7 +255,35 @@
       "texto": "epicmax.co",
       "url": "https://epicmax.co/"
     },
-    "platinum": true,
+    "platinum": false,
     "contact": "hello@epicmax.co"
+  },
+  {
+    "name": "HeroDevs",
+    "logo": "herodevs.png",
+    "flipLogo": true,
+    "hero": "herodevs-hero.png",
+    "intro": "El fin de ciclo de vida (EOL) de las versiones de software de código abierto crea grandes desafíos para las empresas. HeroDevs ofrece dos soluciones: (1) Proporcionamos soporte continuo para software de código abierto que ha llegado al final de su vida útil, y (2) brindamos servicios de consultoría e ingeniería para ayudar a las empresas a migrar a software de código abierto que tiene soporte",
```

Comment:
no sabía si mantener el anglicismo como end-of-life y open source

Reply:
Asi traducido queda bien, se lee de manera mas natural.